README.md

```diff
@@ -214,6 +214,94 @@ oc apply -f deploy/olm-catalog/subscription.yaml
 ```
 
 ### Creating a Windows MachineSet
+
+#### Pre-configuration steps in case of vSphere cluster:
+##### Windows VM assumptions:
+- We tested using updated version of Windows Server 1909 VM image and corresponding container images
+- We assume that the Windows image is configured with SSH so that WMCO can access it. Following are steps to comfigure SSH
+  - Configure SSH on the Windows VM using the [powershell script](https://gist.githubusercontent.com/ravisantoshgudimetla/087bf49088f70edf7f97777386d12f18/raw/0f13048697130a0265ed1050591361fdebcee3ae/powershell.yaml)
+  - Create a new file called C:\Users\Administrator\.ssh\authorized_keys and copy your public key to this file
+- We assume that the VMWare tools has been installed on the Windows VM
+- We assume that the [tools.conf](https://docs.vmware.com/en/VMware-Tools/11.1.0/com.vmware.vsphere.vmwaretools.doc/GUID-EA16729B-43C9-4DF9-B780-9B358E71B4AB.html) at C:\ProgramData\VMware\VMware Tools\tools.conf is having an entry for
+  -`exclude-nics=`
+  
+ - This is to ensure that tools won't ignore the cloned vNIC generated on the Windows VM by hybrid-overlay and the VM has an IP Address in the vCenter.
+ Also, please ensure that the tools Windows service is running in the Windows VM.
+
+- We assume that the Windows image has been sysprep'ed and it is available for cloning using the following command:
+`C:\Windows\System32\Sysprep\sysprep.exe /generalize /oobe /shutdown /unattend:<path_to_unattend.xml>`
+
+
+- Generally, ignition config files are available on the internal API Server URL in case of IPI however in case of 
+vSphere, the ignition files are available on both internal and external API Server URLs. WMCO as of now, downloads the
+ignition files from internal API server. So, add a new DNS entry for `api-int.<cluster-name>.domain.com` which points
+to external API server URL `api.<cluster-name>.domain.com`. The external api should have been created as part of 
+[cluster install](https://docs.openshift.com/container-platform/4.5/installing/installing_vsphere/installing-vsphere-installer-provisioned.html)
+We will remove this requirement in future
+
+vSphere Windows VMs are cloned off an existing template or VM, for every VM following steps need to be done. We assume sysprep has been run on the VM:
+- Change the hostname of the Windows VM to match Windows Machine you created, following powershell command can be used:
+```
+Rename-Computer -NewName <machine_name> -Force -Restart
+```
+We are trying to automate the above steps and check this page to get latest updates.
+
+Sample vSphere MachineSet:
```

Comment:
In a previous design meeting we decided that we should have a single example MachineSet within the README and link out to different docs for other providers. I think we decided to have vSphere be that single example. In that case we should move the azure example out of the doc and into a seperate doc file, like we did the AWS example